build.go

```diff
@@ -52,32 +49,20 @@ var (
 // https://github.com/SynoCommunity/spksrc/wiki/Synology-and-SynoCommunity-Package-Architectures
 // https://github.com/SynologyOpenSource/pkgscripts-ng/tree/master/include platform.<PLATFORM> files
 var synPlat = map[string][]string{
-	"amd64": []string{"x86_64"},
-	"386":   []string{"i686"},
-	"arm64": []string{"armv8"},
-	"arm/5": []string{"armv5", "88f6281", "88f6282"},
-	"arm/7": []string{"armv7", "alpine", "armada370", "armada375", "armada38x", "armadaxp", "comcerto2k", "monaco", "hi3535"},
+	"amd64": {"x86_64"},
+	"386":   {"i686"},
+	"arm64": {"armv8"},
+	"arm": {
```

Comment:
update the comment then to delete the "or GOARCH/GOARM" part.

are you going to add a second map to map from SynoArch to GOARM value?

Reply:
made it `arm/5` and `arm/7` again. also using the value to fill out `GOARM` now.